connect/mirror/src/main/java/org/apache/kafka/connect/mirror/MirrorSourceTask.java

```diff
@@ -316,13 +315,14 @@ static class PartitionState {
 
         // true if we should emit an offset sync
         boolean update(long upstreamOffset, long downstreamOffset) {
-            long upstreamStep = upstreamOffset - lastSyncUpstreamOffset;
-            long downstreamTargetOffset = lastSyncDownstreamOffset + upstreamStep;
+            // This value is what OffsetSyncStore::translateOffsets would compute for this offset given the last sync.
+            // Because this method is called at most once for each upstream offset, simplify upstreamStep to 1.
```

Comment:
This comment mentions `upstreamStep` but we no longer have a field named that.

Is this line really necessary? Do you think it would be clear enough with just the comments on lines 318 and 320?